src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -408,7 +414,7 @@ const CurationPage = (props: ICurationPageProps) => {
 };
 
 const mapStoreToProps = ({ geneStore, firebaseGeneStore, firebaseMetaStore, firebaseDrugsStore, authStore }: IRootStore) => ({
-  findAllGeneEntities: geneStore.findAllGeneEntities,
```

Comment:
We can remove `findAllGeneEntities` from gene store since it does the same as the `searchEntities` method.

Reply:
I will update in my audit PR.